chore: APP-2148: switch from nsp to yarn audit

svlapin opened the pull request:
chore: APP-2148: switch from nsp to yarn audit

adieuadieu commented:
Side comment: Should we update the pipeline config for CircleCI in `~/.circleci` to include a security-check step such that this repo is in line with our others?

svlapin commented:
@adieuadieu addressed your comment - as circle node images come with yarn preinstalled, I only had to ensure it has the latest version and added the step. Please have another look.

svlapin commented:
@adieuadieu also it probably makes sense to set `ci/circleci: security-check` as required
